learning_resources/etl/utils.py

```diff
@@ -690,3 +693,12 @@ def parse_certification(offeror, runs_data):
             if (availability and availability != AvailabilityType.archived.value)
         ]
     )
+
+
+def clean_data(data: str) -> str:
+    """Remove unwanted html tags from text"""
+    if data:
+        return nh3.clean(
+            data, tags=ALLOWED_HTML_TAGS, attributes=ALLOWED_HTML_ATTRIBUTES
```

Comment:
Should we add `clean_content_tags` to remove tag contents where they don't make sense without the tag behavior? 

Thinking e.g.
```
Please see more detail <a href="http://url.com">here</a>
```
is better as `"Please see more detail"` than `"Please see more detail here"`.

Also may authors attempting to add styles?

In the tests the <style> contents are also inside html tags (actually xml CDATA), so are getting removed.

If we had plain styles, e.g.

```
<style>body: {background: blue;}</style>
```

We probably want to remove completely rather than displaying the contents.